extension/page-extractor.js

```diff
@@ -29,9 +23,22 @@ async function extractEntities() {
     pageEntity.image = linkImage.href;
   }
   results.push(pageEntity);
+  results = results.concat(extractManifests());
   return results;
 }
 
+// This is out in a separate function, as extractMicrodata() may be replaced
+// with a library for https://github.com/PolymerLabs/arcs/issues/431
+function extractManifests() {
+
+  return Array.prototype.map.call(
+    document.querySelectorAll('link[type="'+manifestType+'"]'),
+    function(link) {
```

Comment:
This looks like an opportunity for new exciting arrow functions like (link) => { return {...} }